package.json

```diff
@@ -1,11 +1,18 @@
 {
   "scripts": {
     "test": "xo && ava",
+    "build": "cd src && each '{*.js,!(libs)/**/*.js}' --parallel 10 -- rollup --config node:webext $PWD/%p --output $PWD/../extension/%p",
```

Comment:
No no the files that `each` is looking for is are currently just three. Content, background and options. The "clean" alternative 3 separate build steps or an alternative looping tool that just processes as many files as it finds. Unfortunately rollup does not support multiple builds in one call